cmd/certsuite/upload/results_spreadsheet/sheet_utils.go

```diff
@@ -0,0 +1,161 @@
+package resultsspreadsheet
+
+import (
+	"fmt"
+
+	"google.golang.org/api/sheets/v4"
+)
+
+func getHeadersAsInterfaceList(sheet *sheets.Sheet) []interface{} {
```

Comment:
`getHeaders` is good enough, and you should document here why you need to return a `[]interface{}` instead of the straightforward `[]string`

Reply:
I'll change this function to two different functions that will return `[]string`. Those functions will handle the different types that the headers need to be extracted from.